nixos/modules/hardware/video/hidpi.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, pkgs, config, ...}:
+with lib;
+
+{
+  options.hardware.video.hidpi.enable = mkEnableOption "Font/DPI configuration optimized for HiDPI displays";
+
+  config = mkIf config.hardware.video.hidpi.enable {
+    console.font = lib.mkDefault "${pkgs.terminus_font}/share/consolefonts/ter-i32n.psf.gz";
```

Comment:
After finally reading terminus's readme, I've discovered we probably want `ter-v32n.psf.gz` for maximum character coverage:

```

2.4. Legend.

names   mappings                covered codepage(s)

ter-1*  iso01, iso15, cp1252    ISO8859-1, ISO8859-15, Windows-1252
ter-2*  iso02, cp1250           ISO8859-2, Windows-1250
ter-7*  iso07, cp1253           ISO8859-7, Windows-1253
ter-9*  iso09, cp1254           ISO8859-9, Windows-1254
ter-c*  cp1251, iso05           Windows-1251, ISO8859-5
ter-d*  iso13, cp1257           ISO8859-13, Windows-1257
ter-g*  iso16                   ISO8859-16
ter-h*  cp1255, iso08           Windows-1258, ISO8859-8
ter-i*  cp437                   IBM-437
ter-k*  koi8r                   KOI8-R
ter-m*  mik                     Bulgarian-MIK
ter-p*  pt154                   Paratype-PT154
ter-u*  koi8u                   KOI8-U

ter-v*  all mappings / code pages listed above and many others, about 110
        language sets, 8 or 16 foreground colors depending on the kernel and
        console driver versions

names   weight

ter-*n  normal
ter-*b  bold
ter-*v  CRT VGA bold
```

Reply:
The `v` font would be ideal for character set coverage, but it looks like it'd reduce the colour functionality of the Linux console even further; [ArchWiki says](https://wiki.archlinux.org/index.php/Linux_console):

> The Linux console uses UTF-8 encoding by default, but because the standard VGA-compatible framebuffer is used, a console font is limited to either a standard 256, or 512 glyphs. If the font has more than 256 glyphs, the number of colours is reduced from 16 to 8.

Reply:
I'm not sure, I just tested
```
for i in {0..8}; do tput setaf $i ; printf "% 2d" $i ; done
tput bold; for i in {0..8}; do tput setaf $i ; printf "% 2d" $i ; done
```
on the console with `ter-v32n` and `ter-132n` and got 16 colours in both cases. Maybe this doesn't apply anymore, or it depends on drivers?

Reply:
If your device matches `/sys/class/graphics/fb[0-9]+` it's the framebuffer driver and not VGA. I would assume that all new monitors would use a framebuffer driver rather than VGA.

Reply:
I switched to `ter-v32n`. If it is causing problems, we can change it.